common/src/main/java/org/tron/common/parameter/CommonParameter.java

```diff
@@ -177,21 +180,21 @@ public class CommonParameter {
   @Getter
   @Setter
   public String nodeExternalIp;
-  @Getter
-  @Setter
-  public boolean nodeDiscoveryPublicHomeNode;
-  @Getter
-  @Setter
-  public long nodeDiscoveryPingTimeout;
-  @Getter
-  @Setter
-  public long nodeP2pPingInterval;
+//  @Getter
+//  @Setter
+//  public boolean nodeDiscoveryPublicHomeNode;
+//  @Getter
+//  @Setter
+//  public long nodeDiscoveryPingTimeout;
+//  @Getter
+//  @Setter
+//  public long nodeP2pPingInterval;
```

Comment:
Why is it commented out instead of just deleting the code?

Reply:
We could delete it next release version.

Reply:
> Why not just delete it? 

\+ 1, git ensures that changes are kept.

Reply:
ok，I will delete it